conftest.py

```diff
@@ -0,0 +1 @@
+# empty to allow importing pytest
```

Comment:
Got a source on why/how this is necessary? I know we've got blank __init__ files but I'd like to get rid of those too. It's a student pedagogy thing, reduce the chance they get distracted by something that's "just the way python works" if possible.

If changing working directory or doing something funky one-time in a test fixture gets rid of the empty file, I think that's the better pedagogy tradeoff.